cityofaustin/atd-vz-data
# Allow multiple related record tables to be expanded at the same time

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17560

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Follow instructions in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
2. Navigate to a crash, see that you can now open all the related records tables at the same time. They should all default to open. Test editing fields.

---
#### Ship list
